starter_code/views/moviedetail.hbs

```diff
@@ -0,0 +1,10 @@
+<h1>{{ title }}</h1>
+<div>
+    <img src="{{image}}" alt="">
+</div>
+    <h3> {{ director }}</h3>
+    <h4> {{ stars }}</h4>
```

Comment:
You can use an each loop when you're dealing with an array: `<ul>{{#each stars}}<li>{{this}}</li>{{/each}}</ul>`